backend/infrahub/config.py

```diff
@@ -586,6 +586,7 @@ class AnalyticsSettings(BaseSettings):
 class ExperimentalFeaturesSettings(BaseSettings):
     model_config = SettingsConfigDict(env_prefix="INFRAHUB_EXPERIMENTAL_")
     graphql_enums: bool = False
+    node_mutated_event: bool = True
```

Comment:
Will this be used for anything? Seems like we only add it in an if-statement for now and that it pretty much just keeps the old behaviour?

Reply:
It was there to be disabled on purpose if we wanted to check performance diff when its disabled.
As-per the internal slack thread, I have removed this commit from the PR so that we rather work on improving the to_graphql call for events (removing properties).